config/jobs/kubernetes/sig-testing/typecheck.yaml

```diff
@@ -23,10 +23,10 @@ presubmits:
         resources:
           limits:
             cpu: 5
-            memory: 32Gi
+            memory: 16Gi
```

Comment:
This seems a bit low, considering even the 32/8 core machine has 4 gigs per core and typecheck is in fact memory intensive? We should not aim to use 100% of system memory but we can do say 18gi.

We're usually CPU bound for autoscaling anyhow (IE no jobs are requesting memory : cores in excess of the core : memory ratio of the host)

Reply:
Same for other jobs like this one.

Reply:
actually given we won't use all cores we can do 1:1 with host ratio, which would be 20 gi.

Reply:
https://monitoring-eks.prow.k8s.io/d/96Q8oOOZk/builds?orgId=1&refresh=30s&var-org=kubernetes&var-repo=kubernetes&var-job=pull-kubernetes-typecheck&var-build=All

I can make this one 20